web/src/compose_ui.ts

```diff
@@ -32,7 +32,7 @@ type ComposeTriggeredOptions = {
 };
 type ComposePlaceholderOptions = {
     direct_message_user_ids: number[];
-    message_type: messageType;
+    message_type: MessageType;
     stream_id: number;
     topic: string;
 };
```

Comment:
Probably also this type should be changed to a union type where which of `direct_message_user_ids`  and `stream_id` is undefined depends on the message type.

Reply:
Updated `ComposeTriggeredOptions` to do that,and then also updated `ComposePlaceholderOptions`